mirrord-layer/src/config.rs

```diff
@@ -29,6 +29,9 @@ pub struct LayerConfig {
     #[envconfig(from = "MIRRORD_AGENT_TTL", default = "0")]
     pub agent_ttl: u16,
 
+    #[envconfig(from = "MIRRORD_AGENT_STEAL_TRAFFIC", default = "false")]
```

Comment:
Hmm maybe add consistency between names, and have it be `TCP_STEAL_TRAFFIC` (also apply the same namimg everywhere).